back/graphql/query/query.js

```diff
@@ -1,49 +1,76 @@
-import { GraphQLInt, GraphQLObjectType, GraphQLString } from "graphql";
-import { PersonType } from "../data_types/dataTypes.js";
-import { query } from "../../db/query.js";
+import {
+  GraphQLInt,
+  GraphQLObjectType,
+  GraphQLList,
+  GraphQLString,
+  GraphQLNonNull,
+} from "graphql";
+import { NewsType } from "../data_types/dataTypes.js";
+import { conn } from "../../db/connection.js";
 
-export const Query = new GraphQLObjectType({
-  name: "query", // 为了维持语义，没有实际作用
+async function getNewsById(id) {
+  let result;
+  if (id === undefined) {
+    result = await conn.execute("SELECT * FROM news");
+  } else {
+    result = await conn.execute("SELECT * FROM news WHERE id = ?", [id]);
+  }
+
+  const [row, _fields] = result;
```

Comment:
Please expand on this. What is `row` and `_fields`.

Reply:
`rows` is the date. `fields` is the meta date of each column (for example, the type / the name / and so on).

It is typo that I using `row` rather than `row`. I will fix it.